1/debian-9/rootfs/app-entrypoint.sh

```diff
@@ -15,21 +15,40 @@ echo "Starting application ..."
 if [ "$1" = "/run.sh" ]; then
   # Create a Symfony app if not found
   if [ ! -d "$PROJECT_DIRECTORY" ] ; then
-    log "Creating example Symfony application"
-    nami execute symfony createProject --databaseServerHost "$MARIADB_HOST" --databaseServerPort "$MARIADB_PORT_NUMBER" --databaseAdminUser "$MARIADB_USER" "$SYMFONY_PROJECT_NAME" | grep -v undefined
+    log "Creating a project with symfony/skeleton"
+
+    composer create-project symfony/skeleton $SYMFONY_PROJECT_NAME
+
+    if [ -z "$SYMFONY_NO_DB" ] ; then
+      log "Installing symfony/orm-pack"
+      composer require symfony/orm-pack -d $PROJECT_DIRECTORY
+    fi
+
+    export DATABASE_URL=mysql://$MARIADB_USER@$MARIADB_HOST/$MARIADB_DATABASE
+
+    if [ ! -f "$PROJECT_DIRECTORY/.env.local" ] ; then
+      touch $PROJECT_DIRECTORY
```

Comment:
I think the path needs to be completed:

```suggestion
      touch $PROJECT_DIRECTORY/.env.local
```